### ISSUE TITLE
Light Showing Through Tiles

### ISSUE THREAD

**Issue by Djeryv** [ClassicUO/ClassicUO#1327]
There are light sources under the water tiles (not statics) in this picture, but they are glowing through the tiles. They do not glow through statics though so that is fine...just the land/sea tiles.

![image](https://user-images.githubusercontent.com/49735284/114272494-01faca80-99dc-11eb-94ab-5129f5a260be.png)

**Comment by gaechti** [comment]
dunno, imho this looks awsome like this

**Comment by Djeryv** [comment]
Yeah...but if you have caves underground with torches it just looks odd and out of place when running on the surface of the land.
